mqtt_esp8266_umbrella/mqtt_esp8266_umbrella.ino

```diff
@@ -30,20 +43,37 @@ const char* client_id = CONFIG_MQTT_CLIENT_ID;
 
 const char* mqtt_topic = CONFIG_MQTT_TOPIC;
```

Comment:
Why are some variables `snake_case` and others `camelCase`? Standardize on one, and maybe run it through a linter for good measure.

Reply:
I'm going to look more into using an actual IDE for this. PlatformIO looks really neat, but I don't want to change things out for that yet. In the meantime, 👍 on standardizing on `camelCase` throughout.